# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1541097507 14400
#      Thu Nov 01 14:38:27 2018 -0400
# Node ID 94fecd437d9ca241bec96d4cf8a82daa9045a7d8
# Parent  c60a5493688aa619921bd00dd925db033bc8a104
* pt-eval.cc: Fix call to JIT evaluate function.

diff --git a/libinterp/parse-tree/pt-eval.cc b/libinterp/parse-tree/pt-eval.cc
--- a/libinterp/parse-tree/pt-eval.cc
+++ b/libinterp/parse-tree/pt-eval.cc
@@ -1419,17 +1419,17 @@ namespace octave
             args = args.slice (1, args.length () - 1, true);
           }
         else
           panic_impossible ();
       }
 
 #if defined (HAVE_LLVM)
     if (user_function.is_special_expr ()
-        && tree_jit::execute (*this, args, retval))
+        && tree_jit::execute (user_function, args, retval))
       return retval;
 #endif
 
     unwind_protect frame;
 
     // XXX FIXME
     frame.add_method (user_function, &octave_user_function::set_call_depth,
                       user_function.call_depth ());
